src/panels/config/lovelace/resources/ha-config-lovelace-resources.ts

```diff
@@ -189,6 +189,8 @@ export class HaConfigLovelaceRescources extends LitElement {
             text: this.hass!.localize(
               "ui.panel.config.lovelace.resources.refresh_body"
             ),
+            confirmText: this.hass.localize("ui.common.yes"),
+            dismissText: this.hass.localize("ui.common.no"),
```

Comment:
Nit: I would prefer "Refresh" / "Ignore", because it makes it clear what the buttons do if you don't read the text. 🤷

Reply:
I would be fine with "Refresh", but I am not sure about "Ignore". In other places such as the integration config page, "Ignore" means really ignore from now on out = do not ask again. That however is not the case here and therefore could be confusing plus the user is not really ignoring it here (in the sense of stopping e.g. a longer guided "wizard" / dialog), but deciding "no" right on this popup.

Reply:
I like "Refresh" for the yes button. The no could be something as `Not now`

Reply:
I changed to "Refresh".

@bramkragten I am not sure if there really is a meaningful difference between "No" and "Not now". Isn't a "No" basically always implicitly a "Not now"? There are not many situations where you couldn't come back later and do whatever you previously declined. If there is no real difference I would stick with "No" since shorter and commonly (and consistently) used through the frontend today.

Reply:
@spacegaier I myself was influenced by the [Apple Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/views/alerts/), which say that you should avoid using Yes and No for button titles.

You point out that No is already widely used in the frontend, but hey, it's never too late to start doing better :D

I vote for "Not Now" myself, because it is clear what the dialog will do even if you *only* read the button titles.

Reply:
I now went through the code base and applied this principle of more meaningful button texts to all dialogs. Got rid of yes/no in most places and substituted by clearer texts, e.g. delete/cancel or enable/cancel. The concrete refresh topic that started this PR is no refresh/not_now.

Let me know what you think.